#[Harvard Chan Bioinformatics Core](http://bioinformatics.sph.harvard.edu/training/)

*Based on the NGS Analysis Course [wiki](http://tinyurl.com/hbc-ngscourse-page)*

## People
* Shannan Ho Sui
* Brad Chapman
* Rory Kirchner
* Meeta Mistry
* Radhika Khetani
* Mary Piper
* Lorena Pantano
* others

## Contact

* consult e-mail: <bioinformatics@hsph.harvard.edu>
* training e-mail: <hbctraining@hsph.harvard.edu>
* website: <http://bioinformatics.sph.harvard.edu/training/>
* twitter: <@bioinfocore>

## Trainees
* Sarah DeVos
* Pankaj Mandal
* Kevin King
* Iliana Soto
* Chen Lossos
* Carman Li
* Laura Lascarez-Lagunas
* Ahilya Sawh
* Alexander Naydich
* Anrea Yung
* Chengyu Zou
* Emily Egan
* Joseph Klim
* Mark Zimmerman
* Tao Lu
* Yuvabharath Kondaveeti
* Bagriel Berriz
* Dejan Maglic
* Suzanne Hickman

## Notes 

Human Genome Project (1990-2003)

HapMap Projects: Map the haplotype diversity in the human genome

* identify 8 million SNP

1,000 Genomes Project (2008-2015)

* 4x sequencing of 2,504 samples
* SNPs (single nucleotide polymorphisms)
* larger variants (500,000 bp long)

UK10K Consortium


Tumor versus wildtype sample sequencing specs

* turnaround time: 1 week
* cost: $30,000

hg37

* 23 chromosomes
* 3.5 billion bp's
* 20,000 coding genes



GRCz10 is the 10th zebrafish reference genome

* 26 chromosomes (including mitochodrion)
* 1.5 billion bp's
* 25,000 coding genes




## Literature

Nature Reviews Genetics 2, 573-583 (August 2001)

Nature 470, 198–203 (10 February 2011)

*Lander, E. S. Initial impact of the sequencing of the human genome. Nature doi:10.1038/nature09792*

*Harismendy, O. et al. Population sequencing of two endocannabinoid metabolic genes identifies rare and common regulatory variants associated with extreme obesity and metabolite level. Genome Biol. 11, R118 (2010)*

*Korbel, J. O. et al. Paired-end mapping reveals extensive structural variation in the human genome. Science 318, 420–426 (2007)*

*Chen, K. et al. BreakDancer: an algorithm for high resolution mapping of genomic structural variation. Nature Methods 6, 677–681 (2009)*

*Alkan, C. et al. Personalized copy number and segmental duplication maps using next-generation sequencing. Nature Genet. 41, 1061–1067 (2009)*

*Sudmant, P. H. et al. Diversity of human copy number variation and multicopy genes. Science 330, 641–646 (2010)*

*Durbin, R. M. et al. A map of human genome variation from population-scale sequencing. Nature 467, 1061–1073 (2010)*

*Ley, T. J. et al. DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome. Nature 456, 66–72 (2008)*

*Birney, E. et al. Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. Nature 447, 799–816 (2007)*

*Green, E. D., Guyer, M. S. & National Human Genome Research Institute Charting a course for genomic medicine from base pairs to bedside. Nature doi:10.1038/nature09764*

*Loh, J. et al. Detection of novel sequences related to African Swine Fever virus in human serum and sewage. J. Virol. 83, 13019–13025 (2009)*

*Presti, R. M. et al. Quaranfil, Johnston Atoll, and Lake Chad viruses are novel members of the family Orthomyxoviridae. J. Virol. 83, 11599–11606 (2009)*

*Finkbeiner, S. R. et al. Identification of a novel astrovirus (astrovirus VA1) associated with an outbreak of acute gastroenteritis. J. Virol. 83, 10836–10839 (2009)*

*Wheeler, D. A. et al. The complete genome of an individual by massively parallel DNA sequencing. Nature 452, 872–876 (2008)*

*Lupski, J. R. et al. Whole-genome sequencing in a patient with Charcot-Marie-Tooth neuropathy. N. Engl. J. Med. 362, 1181–1191 (2010)*

*Roach, J. C. et al. Analysis of genetic inheritance in a family quartet by whole-genome sequencing. Science 328, 636–639 (2010)*

*Jones, S. J. et al. Evolution of an adenocarcinoma in response to selection by targeted kinase inhibitors. Genome Biol. 11, R82 (2010)*

*Ng, S. B. et al. Exome sequencing identifies the cause of a mendelian disorder. Nature Genet. 42, 30–35 (2010)*

*Ng, S. B. et al. Exome sequencing identifies MLL2 mutations as a cause of Kabuki syndrome. Nature Genet. 42, 790–793 (2010)*

*Gilissen, C. et al. Exome sequencing identifies WDR35 variants involved in Sensenbrenner syndrome. Am. J. Hum. Genet. 87, 418–423 (2010)*

*Chin, C. S. et al. The origin of the Haitian cholera outbreak strain. N. Engl. J. Med. 364, 33–42 (2011)*

This material has been adapted from a lesson developed by members of the teaching team at the [Harvard Chan Bioinformatics Core](http://tinyurl.com/hbc-ngscourse-page) (HBC). These are open access materials distributed under the terms of the [Creative Commons Attribution license (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.

The materials used in this lesson were derived from work that is Copyright © Data Carpentry (<http://datacarpentry.org/>). All Data Carpentry instructional material is made available under the [Creative Commons Attribution license (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/).
Adapted from the lesson by Tracy Teal. Original contributors: Paul Wilson, Milad Fatenejad, Sasha Wood and Radhika Khetani for Software Carpentry (<http://software-carpentry.org/>)

[https://wiki.galaxyproject.org/Learn/GalaxyNGS101](https://wiki.galaxyproject.org/Learn/GalaxyNGS101)








